### DISCUSSION TITLE
Incentivizing small farmers

### DISCUSSION THREAD

**Discussion by dpwiener** [Chia-Network/chia-blockchain#6678] [Ideas]
No good deed goes unpunished.  I propose herein a good-deed anti-punishment strategy for Chia.

As originally envisioned, Chia's Proof of Space and Time architecture had the potential to democratize the blockchain validation process by utilizing idle memory capacity spread over a large number of computer systems.  This not only had a David-vs-(Bitcoin)-Goliath vibe, it offered buy-in and participation by a huge audience of users, who in turn would evangelize the new crypto currency and encourage the invention of applications using its Chialisp programming language.

But Chia's launch was too successful: The price of a Chia coin skyrocketed, which in turn offered potential riches to large Chia farmers.  Those riches turned out to be illusory.  The Chia network grew so fast that only enormous quantities of hard drive space gave farmers any hope of winning Chia coins and then of keeping up with the overall network expansion rate.

The little guys with one or several terabytes of spare hard drive memory have essentially no chance of successfully farming Chia coins.  They might as well buy lottery tickets; their odds of winning are similar.  That has created a backlash among early adopters who would otherwise be Chia's strongest supporters.

Farming rewards are set at 3,363,840 coins per year for the first three years, which have a total annual value of over $1.5 billion at current market rates.  The Chia company has a pre-farm strategic reserve of 21 million coins.  I propose that the Chia company separately incentivize small farmers using its strategic reserve.  A possible scheme would be as follows:

- Any farmer could voluntarily register identifying information (name, address, email, wallet address) with the Chia company, along with a computer model and serial number, and ID information for up to ten plots (about one terabyte) which are being farmed by that computer.

- The farmer would run a utility program which would report to the Chia company the amount of time each of the plots was spending farming.

- The farmer's combined farming time for that limited set of plots, taken as a percentage of the combined farming time of all small farmers, would be that farmer's probability of winning an award.

- The Chia company would contribute some number of coins (e.g., up to half a million per year out of its strategic reserve) to reward small farmers.

- At periodic intervals (e.g., daily) the Chia company would randomly select winners using the above probabilities, and distribute fractional coin rewards (e.g., one-tenth coins) to some set number of winners.  This would be independent of a farmer's chance of winning Chia coins in the normal manner.

- This small farmer incentive system would remain at the total discretion of the Chia company, which could cut off any farmers who were suspected of gaming the system.

- Small farmers could all expect to receive occasional (typically several months?), albeit small, rewards.  Those rewards would not be large enough to encourage widespread cheating (e.g., buying multiple computers and registering them under fake names).  But the rewards would be sufficient to incentivize many personal computer owners to create between one and ten plots (depending upon their available memory) and farm them as background tasks.

The end result would be a large, supportive user base with a heightened interest in using their coins and Chialisp applications.  This broader base would produce a more robust Chia eco-system and likely push up the value of Chia coins, resulting in an increase in the value of the company's remaining strategic reserve to more than compensate for the reward expenditures.  It would also show that the company was pro-actively attempting to mitigate Chia's impact on the SSD and hard drive supply chains.  Implementing this proposal (or some variant) would be well worth it from a PR perspective alone.

Daniel Wiener

**Comment by room101-dev** [comment]
Surely rose glasses have infected your brain.

The Chinese own and run Chia since day one last year [2020] when the funded the project, or I should say 'rescued' chia [ a floundering going no where project lost since 2017 ], the deal is this.

The Chinese billionaires ( same guy who funded the ukraine bitcoin-gold, aka bch ) wanted

1.)  IOU's ran by chinese exchanges they sent the price to $1600 usd, pre main-net, the west had nothing to do with chia price, pre mainnet launch 5may2021
2.) The xch 21M pre-mine is the property of the 'chinese investor'
3.) The dev-team in SV aka bram&hoffman get the IPO, said to be wroth in access of $10B USD
4.) The chinese  people get to pre-mine on hpool, the west told to 'wait', the pools were to never come
5.) The chinese people get to play 'lottery' with the IOU system

7.) The HDD companys all Asian based like WD, Samsung, ... all were working since last year [2020]  on this deal, that like GPU's the HDD's to go 10x in price, in the west we would call this 'collusion', in asia its business as normal

8.) The western gimps, hairlips, and believers get zip, zilch nada chia. They aren't even bag-holders, they're empty bag holders,and then they're told to re-plot. The world has never seen a bigger pool of lemmings.

That's the deal, now you adjust your polly-anna wishes to reality.

**Reply by dpwiener** [reply]
![image](https://user-images.githubusercontent.com/85005633/121639988-37ef1600-ca42-11eb-8ab5-7d764d15b3b4.png)

**Reply by joelrb** [reply]
Figment of his imagination. No basis in reality. Conspiracy theories. Fallacious arguments. Ignore unless real proof or hard evidence is presented.

**Comment by pluto2021** [comment]
a better way to Incentivizing small farmers is to get on the IPO bus. We need to be united to fight for our right to be part of this IPO deal! We make the network space, and we deserve part of the pie!!!

**Reply by dpwiener** [reply]
Nobody has a "right" to be part of an IPO.  When Chia goes public, at that time you'll have the opportunity to invest in the stock (which may go up or down, like any stock).  But that has nothing to do with incentivizing small farmers.

I continue to think that the incentive program I suggested above (or a variant) would promote Chia and make it a more widely-accepted cryptocurrency.  That in turn would add significant value to the company for its anticipated IPO.

**Reply by pluto2021** [reply]
I would highly suggest some reading on how Wall Streets really plays an IPO. They are the financial industry mafia, literally! If you buy at IPO, you may or may not get the 5% leftovers, if not a bloody nose. They are playing asymmetric risk/rewards, while you are counting on your luck.

**Reply by dpwiener** [reply]
I'm aware of how IPOs work.  It's usually a bad idea to buy into IPOs immediately after they go public.  Typically they will spike up, then drop back down, often to their opening price or lower.  That would be a better time to buy the stock, if you are convinced that it has long-term appreciation potential.  The other route is if a company goes public through a SPAC, which I understand that Chia is considering.  In that case you can often buy the SPAC stock at close to it's nominal price of $10, and then wait for the reverse merger to be completed, effectively getting in as close as possible to the ground floor.  It still means that the early venture capital investors and company management will reap the most rewards, but I consider that reasonable since they took the greatest risks.

**Reply by pluto2021** [reply]
Well, for Chia netspace, besides the Chia team and private investors, farmers are also sharing the risks. I wonder how many farmers have recovered their investment only through the XCH rewards? If you are still holding on to all farmed XCH, with the current network size, you need to be able to get your per plotted TiB cost down to <$15 to break even on average, if you managed to farm proportional amount of XCH based on your space contribution. If farmers are sharing the risks, why this risk is not rewarded through this IPO? If this XCH keeps diving down, the net loss would keep increasing, yet, no predictable rewards associated. It would be a net loss for the farmers. Yet, the Wall Streets will try to pump up the valuation regardless, although affected by the XCH pricing, and the risks taking by the Chia team and private investors will be rewarded big regardless, leaving farmers with empty plates and high probability of a net loss.

**Reply by dpwiener** [reply]
Pluto2021, you make a reasonable argument.  But farmers should have been able to predict that the initial conditions were unsustainable:  The initial high Chia price combined with the initial low network size seemed to offer an enormous ROI and quick riches.  That couldn't last; supply and demand are always going to tend towards an equilibrium.  Chia prices have to drop and the network has to grow until the profit from farming becomes only slightly greater than the capital investment for the most efficient farmers.

This is the problem that Bitcoin has.  Only the giant mining conglomerates who have invested heavily in specialized equipment and cheap electricity can make a predictable profit, and even that can disappear quickly if the Bitcoin price plunges.  Decentralization offers Chia a partial solution.  Small farmers can use existing spare computer and memory capacity without new investments if the proper incentives are put in place.  No one will get rich by being a small Chia farmer, but it should be possible to earn small rewards commensurate with the utilization of existing underutilized resources.

Full disclosure:  I have a spare computer in my home with about one terabyte of SSD memory, and I'm currently creating Chia plots on it.  I'm doing it out of curiosity, to see what the process is like.  I'm a software engineer, and Chia piqued my interest.  But my projected payback time is decades, and that time will just keep lengthening as the network grows, so I don't expect to ever win any Chia coins (unless I get super-incredibly lucky).  Other people who don't share my curiosity are unlikely to bother to farm on their personal computers as long as the probability of winning any kind of reward is near-zero.

**Comment by ktkiebel** [comment]
If Chia actually wanted to decentralize they would just reduce rewards based on farm size.  They clearly dont care about decentralization.

**Reply by dpwiener** [reply]
My guess is that there are conflicting concerns and trade-offs between encouraging centralized large farmers and decentralized small farmers.  Large farms have the potential to produce large rewards, which lead people to gravitate to the cryptocurrency.  Certainly the impact of Chia farming on hard drives (creating shortages and price increases) generated an enormous amount of free publicity, and immediately established Chia as a serious player in the crypto sphere.  But a parallel effort to incentivize small farmers and a decentralized eco-system should still be feasible.  The Chia company could have its cake and eat it too.  Yum?

**Reply by speedmann** [reply]
> If Chia actually wanted to decentralize they would just reduce rewards based on farm size. They clearly dont care about decentralization.

everytime someone doesn't understand decentralization a cute kitten dies horribly. You just killed another 10 as your statement is enough for 10 people not understanding decentralization...

**Reply by ktkiebel** [reply]
I'd probably also make make stuff up like dying kittens if I had a big farm and couldnt come up with an argument to defend it.

**Reply by PapiDot** [reply]
A cute kitten dies horribly was not because of the cat but the cuteness as a form of power excise by the human.

**Reply by speedmann** [reply]
You got that wrong. I not only have a big farm, i have a *gigantic* farm. Of those 25EiB netspace around 20EiB are mine. But for you i will explain again why you're wrong...

>Centralized: Centered in one point or on the authority of one person, party, etc

So, as long as no single person/entity controls *all* of chia, it *is* decentralized. It doesn't matter if it is 2 or 2 million different people. For chias purpose of beeing decentralized aka. not controlled by Chia Network Inc. only one other independent entity would be enough. It also does not matter if there are several whales or even more small farmes. 
But you already knew all of that and just decided to ignore it and cry about how centralized and bad chia is because it did not make you rich...

**Reply by PapiDot** [reply]
I dare to disagree about the understanding of decentralization. Otherwise Google, Intel, Microsoft etc would be qualitied as "decentralized" corporations because even Bill Gates, for instance, has a rather minor stake in Microsoft.

**Reply by speedmann** [reply]
Do i *actually* have to define what an entity is? An entity is a company like Google, Intel, Microsoft.... For the sake of beeing lazy i just did not write person/entity every time...

Edit: As i did not explcitly stated: An entity is not limited to beeing Google intel or microsoft. I can be any person or company or organization or whatever you want.

Jeez... What a time to be alive where you have to write something like this out...

**Comment by PapiDot** [comment]
"This small farmer incentive system" would make the Chia Company act like an authority. It's more like a welfare than decentralization, IMHO.

**Reply by ktkiebel** [reply]
Can you have a capitalist's crypto thats actually for the people ? ........ not sure.  Whoever comes up with one will probably win the crypto race.  Not looking like Chia at this point.

**Reply by PapiDot** [reply]
That enters the space of philosophical debate. It's equivalent to argue whether an individual could come up with an idea to benefit the society as a whole in particular to weak and poor.

Nonetheless, I agree that Chia scheme in the current form would accomplish nothing but let the rich gets richer.

**Reply by speedmann** [reply]
> Nonetheless, I agree that Chia scheme in the current form would accomplish nothing but let the rich gets richer.

Basically everything does that. Name one single thing which makes the poor richer and the rich less rich.

**Reply by 69chargr** [reply]
> > Nonetheless, I agree that Chia scheme in the current form would accomplish nothing but let the rich gets richer.
> 
> Basically everything does that. Name one single thing which makes the poor richer and the rich less rich.

Not even communism does that. Everyone just gets poorer :-)

**Comment by gromran** [comment]
What did you think? It is the same Ponzi scheme as all crypto "currencies"
